image-specs/el7.3-base

```diff
@@ -0,0 +1,16 @@
+#base=libguestfs:centos-7.3
+#name=el7.3-base
+#osinfo=centos7.3
+#distro=el7
+#version=v1
+#arch=x86_64
+#expand=/dev/sda2
+#lvexpand=/dev/cl/root
+#root-partition=/dev/cl/root
+root-password password:123456
+update
```

Comment:
(Can do in a different patch, but I think this is a good timing) - add:
uninstall iwl*
edit /boot/grub2/grub.cfg:s/set timeout=5/set timeout=0/g